### PR TITLE
[feat]针对要翻译的句子，支持二次选择句子中的某个单词，并进行上下文解释

### PR DESCRIPTION
**Pull request by 100apps**
需求来源：https://www.v2ex.com/t/921195

image

### CODE REVIEW

**Comment by yetone** [comment]
LGTM! Thanks for your contribution!

File: build.mjs [hunk 1]
```diff
@@ -20,7 +20,7 @@ async function esbuildBrowserExtension() {
         minify: true,
         legalComments: 'none',
         sourcemap: true,
-        plugins: [inlineImage()],
+        // plugins: [inlineImage()],
```
Comment (yetone) [inline build.mjs:23]: Why remove this plugin? This will cause packaging failure due to the inability to package gif images.

Comment (100apps) [reply build.mjs:23]: chrome extension，logo image load failure.

Comment (yetone) [reply build.mjs:23]: @100apps Thank you very much for your help in finding this problem and alerting me to it, it was a problem caused by me, I have fixed the problem and pushed to the main branch, I may need to ask you to change this line back and sync the main branch, sorry

Comment (100apps) [reply build.mjs]: @

> @100apps Thank you very much for your help in finding this problem and alerting me to it, it was a problem caused by me, I have fixed the problem and pushed to the main branch, I may need to ask you to change this line back and sync the main branch, sorry

done. thanks

File: src/content_script/translate.ts [hunk 2]
```diff
@@ -50,6 +51,12 @@ export async function translate(query: TranslateQuery) {
                     assistantPrompt = '翻译成简体白话文'
                 }
             }
+            if (query.selectedWords?.length > 0) {
+                // 在选择的句子中，选择特定的单词。触发语境学习功能。
+                systemPrompt = '你是一位英语词义语法专家，你在教我英语，我给你一句英语句子，和这个句子中的一个单词，请用中文帮我解释一下，这个单词在句子中的意思和句子本身的意思,如果单词在这个句子中是习话的一部分，请解释这句句子中的习话，并举几个相同意思的英语例句,并用中文解释例句。如果你明白了请说同意，然后我们开始。'
```
Comment (yetone) [inline src/content_script/translate.ts:56]: Great prompt! And can English be used as the prompt when the target language is not Chinese, and explain words in the target language?

Comment (100apps) [reply src/content_script/translate.ts:56]: good idea.

Comment (100apps) [reply src/content_script/translate.ts]: > Great prompt! And can English be used as the prompt when the target language is not Chinese, and explain words in the target language?

@yetone done

**Pull request merged**